mitxgraders/comparers/comparers.py

```diff
@@ -297,7 +318,25 @@ def vector_phase_comparer(comparer_params_evals, student_eval, utils):
     >>> grader(None, '[2, 2*exp(-i*phi)]')['ok']
     False
 
+    The compaer_params should be list of equal-length vectors:
```

Comment:
Typo in `comparer_params`

Reply:
Also, should be a list containing a single vector, not a list of equal length vectors.